README.md

```diff
@@ -325,6 +325,30 @@ cmake -B build -S . -DBEMAN_EXEMPLAR_BUILD_TESTING=OFF
 
 </details>
 
+<details>
+<summary> Disable `git clone` operations </summary>
```

Comment:
Nit: maybe `Disable pulling Google Test from GitHub` would be a more clear summary here.

Reply:
It's not specifically GitHub that is the issue -- it's the off-box I/O as such.

Is the confusion that it's not obvious that `git clone` is being invoked by `FetchContent`? Possibly we need to update some comments in the `CMakeLists.txt` to make that more clear.